3-0-java-core/src/test/java/com/bobocode/file_stats/FileStatsTest.java

```diff
@@ -0,0 +1,66 @@
+package com.bobocode.file_stats;
+
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class FileStatsTest {
```

Comment:
@maxstasiuk92 you have to put @TestMethodOrder(MethodOrderer.OrderAnnotation.class) on a test class in order to make it work. Have you run the test locally?

Reply:
@tboychuk yes, I did - pass all. I just did not use @Order before and didn't pay attention, that order is different.